packages/web-components/fast-foundation/src/select/select.template.ts

```diff
@@ -23,16 +23,16 @@ export const SelectTemplate: ViewTemplate<Select> = html`
         <div
             aria-activedescendant="${x => (x.open ? x.ariaActiveDescendant : null)}"
             aria-controls="listbox"
-            aria-expanded="${x => x.ariaExpanded}"
             aria-haspopup="listbox"
+            aria-labelledby="selected-value"
             class="control"
             part="control"
-            role="button"
+            role="textbox"
```

Comment:
Axe was looking for a child with the `textbox` role (as per the [combobox spec](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html))

Reply:
Ah - @m4thieulavoie I think Axe is outdated. The combobox from ARIA 1.1 and 1.2 are significantly different. Should this be `combobox` rather than textbox? This doesn't fully support free input of text, so I'm not sure that textbox is accurate. Technically I think this is the select-only combobox: https://w3c.github.io/aria-practices/#examples-2

Reply:
Oh, let me check the axe version. However, the host already has the `combobox` role. So perhaps `button` was the right one here

Reply:
@chrisdholt you were absolutely right. Axe has [this opened issue](https://github.com/dequelabs/axe-core/issues/2505) to add support for the 1.2 combobox.
I'll close this PR as it seems to be a false alarm on our end!

Reply:
All good! Once that AXE update is out, we'd love you to take another pass :)